conda/recipes/dask-cuda/recipe.yaml

```diff
@@ -0,0 +1,59 @@
+# Copyright (c) 2025, NVIDIA CORPORATION.
+schema_version: 1
+
+context:
+  version: ${{ env.get("RAPIDS_PACKAGE_VERSION") }}
+  minor_version: ${{ (version | split("."))[:2] | join(".") }}
+  date_string: '${{ env.get("RAPIDS_DATE_STRING") }}'
+  py_version: ${{ env.get("RAPIDS_PY_VERSION") }}
+  py_buildstring: ${{ py_version | version_to_buildstring }}
+  head_rev: ${{ git.head_rev(".")[:8] }}
+
+package:
+  name: dask-cuda
+  version: ${{ version }}
+
+source:
+  path: ../../..
+
+build:
+  noarch: python
+  string: py${{ py_buildstring }}_${{ date_string }}_${{ head_rev }}
```

Comment:
This seems like a `noarch` package to me, in which case we should remove the python build string from the version string, and also restrict the build to a single Python CI job